File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);
void print_grid(int height);

int main(void)
{
    int height = get_height();
    print_grid(height);
}

int get_height(void)
{
    int size;
    do
    {
        size = get_int("Height: ");
    }
    while (!(size > 0 && size < 9));

    return size;
}

void print_grid(int height)
{
    for (int i = 0; i < height; i++)
    {
        int diff = height - (i + 1);
        // add spaces to left of row
        for (int k = 0; k < diff; k++)
        {
            printf(" ");
        }

        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }

        // add spaces to right of roww
        for (int k = 0; k < diff; k++)
        {
            printf("");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# TODO
height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")

for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1))
